lib/themes/dosomething/paraneue_dosomething/scss/content/_reportback-permalink.scss

```diff
@@ -26,22 +26,23 @@
       border-top: 2px solid $light-gray;
     }
 
-    > .wrapper {
-      display: none;
+    .social-icon {
+      padding: $base-spacing / 2;
 
       @include media($tablet) {
-        display: block;
+        padding: $base-spacing;
       }
     }
+  }
 
-    .social-icon {
-      padding: $base-spacing / 2;
+  .card__copy {
+    .-share {
```

Comment:
It probably wouldn't be an issue, but the danger is if we nested something inside the `.card__copy` that also had a `.-share` modifier, it would also inherit these styles. So better safe than sorry! :smile: